src/server/pachyderm_test.go

```diff
@@ -2222,15 +2229,9 @@ func TestUpdatePipeline(t *testing.T) {
 	// Inspect the first job to make sure it hasn't changed
 	jis, err := c.ListJob(pipelineName, nil, nil)
 	require.Equal(t, 3, len(jis))
-	ji, err := c.InspectJob(jis[0].Job.ID, false)
-	require.NoError(t, err)
-	require.Equal(t, "echo bar >/pfs/out/file", ji.Transform.Stdin[0])
-	ji, err = c.InspectJob(jis[1].Job.ID, false)
-	require.NoError(t, err)
-	require.Equal(t, "echo bar >/pfs/out/file", ji.Transform.Stdin[0])
-	ji, err = c.InspectJob(jis[2].Job.ID, false)
-	require.NoError(t, err)
-	require.Equal(t, "echo foo >/pfs/out/file", ji.Transform.Stdin[0])
+	require.Equal(t, "echo bar >/pfs/out/file", jis[0].Transform.Stdin[0])
+	require.Equal(t, "echo bar >/pfs/out/file", jis[1].Transform.Stdin[0])
+	require.Equal(t, "echo foo >/pfs/out/file", jis[2].Transform.Stdin[0])
```

Comment:
Does this pass? I think `ListJob` doesn't include the `Transform` by default anymore.